django/db/backends/base/base.py

```diff
@@ -432,26 +436,34 @@ def validate_no_broken_transaction(self):
     # ##### Foreign key constraints checks handling #####
 
     @contextmanager
-    def constraint_checks_disabled(self):
+    def constraint_checks_disabled(self, table_names):
```

Comment:
It needs to be optional for backwards compatibility. I can't decide whether it would be better to be optional in master too, or force its usage to specify specific tables to avoid unintentional performance hits.